draft-ietf-rats-architecture.md

```diff
@@ -1124,9 +1127,9 @@ Attestation Result.
 The first approach is to rely on synchronized and trustworthy clocks, and
 include a signed timestamp (see {{?I-D.birkholz-rats-tuda}}) along with the
 Claims in the Evidence or Attestation Result.  Timestamps can also be added on a
-per-Claim basis to distinguish the time of creation of Evidence or Attestation
+per-Claim basis to distinguish the time of generation of Evidence or Attestation
 Result from the time that a specific Claim was generated.  The clock's
-trustworthiness typically requires additional Claims about the signer's time
+trustworthiness can be established via Endorsements and typically requires additional Claims about the signer's time
```

Comment:
Above is probably true, but not necessarily so I'd prefer weakening slightly to perhaps
```suggestion
trustworthiness can generally be established via Endorsements and typically requires additional Claims about the signer's time
```
(rationale: there are odd corner cases not worth mentioning... e.g., where the attesting environment has no clock but does have a secure channel to a peripheral that does - like a GPS receiver, and so it always has to ask the peripheral, so might be more like a composite device or something where the Endorsement of the Attesting Environment itself doesn't help, it's the Endorsement of the other hardware component that does.  Point being vagueness about details is good here.)